test/ably/encoders_test.py

```diff
@@ -10,7 +10,11 @@
 from ably.types.message import Message
 
 from test.ably.restsetup import RestSetup
-from test.ably.utils import BaseAsyncTestCase, AsyncMock
+from test.ably.utils import BaseAsyncTestCase
+try:
```

Comment:
I think it would be nice to test against the python version rather than just catching the import error - that way someone reading this code will be able to see which version it affects and when we can safely remove it

Reply:
Agreed, will do that instead. Both methods are common practice, FYI.